### ISSUE TITLE
Inconsistent testname

### ISSUE THREAD

**Issue by ingobecker** [emberjs/guides#1665]
At the end of the section [The Model Hook](https://guides.emberjs.com/v2.8.0/tutorial/model-hook/) the tutorial reffers to a test named `should initially list 3 rentals`. I guess it's reffering to the test actually named `should list available rentals.` on the page [Setting up Tests](https://guides.emberjs.com/v2.8.0/tutorial/acceptance-test/).

**Comment by toddjordan** [comment]
Good catch!  If you have some time would love help updating that text :-)  if not will get to it soon...

**Comment by ingobecker** [comment]
Should the text be changed or should i make a new screenshot reflecting the names in the text? I also found a minor issue in [Installing Addons](https://guides.emberjs.com/v2.8.0/tutorial/installing-addons/), reffering to an incomplete path for the mirage config which i could fix as well. Should i make changes to the english version only or correct the translations as well?

**Comment by acorncom** [comment]
@ingobecker if you could send in a new screenshot, that'd be great.  Re: your other issue, just make your changes to the english version, the translations sync automatically

**Comment by toddjordan** [comment]
Thanks! make sure you run the screenshot through a program like https://tinypng.com  to ensure fast loading of the image :-)

**Comment by toddjordan** [comment]
also named wrongly in the image in "Services and Utilities".  Part of this fix should include updating that as well.

**Comment by williamhaley** [comment]
@toddjordan I believe I have the appropriate screenshot for the "Services and Utilities" section in this PR https://github.com/emberjs/guides/pull/2129

Please let me know if this is wrong, or if there are any other updates I can make to help out with this issue. Thanks!
